### PR TITLE
Default func_button damage activates spawnflag to on for momentum

### PR DESCRIPTION
**Pull request by braem**
Closes https://github.com/ChaosInitiative/Chaos-FGD/issues/63

### CODE REVIEW

File: fgd/brush/func/func_button.fgd [hunk 1]
```diff
@@ -18,8 +18,10 @@
 		1: "Don't move" : 0
 		32: "Toggle" : 0
 		256: "Touch Activates" : 0
-		512: "Damage Activates" : 0
-		1024: "Use Activates" : 1
+		512: "Damage Activates" : 0 [-MOMENTUM]
+		512: "Damage Activates" : 1 [+MOMENTUM]
+		1024: "Use Activates" : 1 [-MOMENTUM]
+		1024: "Use Activates" : 0 [+MOMENTUM]
```
Comment (Gocnak) [inline fgd/brush/func/func_button.fgd:24]: Are they mutually exclusive? We still want use buttons to work, bhop maps use them for stuff.

Comment (hexaflexahexagon) [reply fgd/brush/func/func_button.fgd:24]: Generally an individual button is going to want one or the other, not both. Also generally momentum buttons tend to use damage more than use so this being default makes the most sense to me. It's always changeable for +use buttons anyways, this doesn't break backwards compatibility or anything

Comment (braem) [reply fgd/brush/func/func_button.fgd:24]: Not really sure on a consensus here - it's good for jump but maybe bad for bhop? Maybe best to leave both on or both off for impartiality?

Comment (Gocnak) [reply fgd/brush/func/func_button.fgd:24]: Leave use on, but make the damage one the momentum conditional

Comment (braem) [reply fgd/brush/func/func_button.fgd:24]: That's good yeah, the use wont actually effect jump at all anyways.

**Review by Gocnak — APPROVED** [review]

**Pull request merged**